.env

```diff
@@ -0,0 +1,2 @@
+TEST=test
+COMPOSE_PROJECT_NAME=test
```

Comment:
do we still need this file in project root? looks like a leftover from ./examples/compose

Reply:
whoops, good catch yes that was from testing